ui/mirage/factories/task-event.js

```diff
@@ -12,5 +12,7 @@ export default Factory.extend({
   exitCode: () => null,
   time: () => faker.date.past(2 / 365, REF_TIME) * 1000000,
 
-  displayMessage: () => faker.lorem.sentence(),
+  // TODO: I think this is probably an archaic property name. See if tests collapse with this removal and decide to remove then.
+  // displayMessage: () => faker.lorem.sentence(),
+  message: () => faker.lorem.sentence(),
```

Comment:
`blocking`: I'm getting 99 test failures. ~97 of them are associated to this diff. Unsure if this is the root of all the test failures...

Reply:
Hi @ChaiWithJai -- please see the code comment TODO in the code here. This was run to see if tests collapse.